ui/apps/platform/src/Containers/PolicyCategories/PolicyCategoriesListSection.tsx

```diff
@@ -1,33 +1,32 @@
 import React, { useEffect, useState } from 'react';
-import { PageSection, Title, Flex, TextInput } from '@patternfly/react-core';
+import { PageSection, Divider, Title, Flex, FlexItem, TextInput } from '@patternfly/react-core';
 
+import { PolicyCategory } from 'types/policy.proto';
 import PolicyCategoriesList from './PolicyCategoriesList';
 import PolicyCategoriesFilterSelect, { CategoryFilter } from './PolicyCategoriesFilterSelect';
+import PolicyCategorySidePanel from './PolicyCategorySidePanel';
 
 type PolicyCategoriesListSectionProps = {
-    // TODO: change once proto goes in
-    // policyCategories: {
-    //     id: string;
-    //     name: string;
-    //     isDefault: boolean;
-    // }[];
-    policyCategories: string[];
+    policyCategories: {
+        id: string;
+        name: string;
+        isDefault: boolean;
+    }[];
+    addToast: (message) => void;
 };
 
-function PolicyCategoriesListSection({ policyCategories }: PolicyCategoriesListSectionProps) {
-    // TODO: to remove once proto is in
-    const allPolicyCategories = policyCategories.map((category) => ({
-        id: category,
-        name: category,
-        isDefault: true,
-    }));
-    const customPolicyCategories = allPolicyCategories.filter(({ isDefault }) => !isDefault);
-    const defaultPolicyCategories = allPolicyCategories.filter(({ isDefault }) => isDefault);
+function PolicyCategoriesListSection({
+    policyCategories,
+    addToast,
+}: PolicyCategoriesListSectionProps) {
+    const [selectedCategory, setSelectedCategory] = useState<PolicyCategory>();
```

Comment:
What do you think about `selectedCategoryId` instead to avoid any problem with object identity?

Reply:
Since any selected category is passed down to the `<PolicyCategorySidePanel>` below, and this will only be a reference to the object, it seems that only storing the ID field would make more work to get the category to pass down, without much performance benefit. Am I missing some other advantage?

Reply:
The question is about potential loss of current selection if object identity changes for selected category.

For example, pages often get all entities after any change. Even if not, after update to selected entity.

Reply:
I think we should defer this until we see how big a problem it would be.